Examples/UIExplorer/js/TextInputExample.android.js

```diff
@@ -623,4 +710,22 @@ exports.examples = [
     title: 'Toggle Default Padding',
     render: function(): ReactElement { return <ToggleDefaultPaddingExample />; },
   },
+  {
+    title: 'Text selection & cursor placement',
+    render: function() {
+      return (
+        <View>
+          <SelectionExample
+            style={styles.default}
+            value="text selection can be changed"
+          />
+          <SelectionExample
+            multiline
```

Comment:
Sorry, I wasn't very clear, and maybe my JS knowledge isn't up to par here.
I'm confused as to what "multiline" does here. Is it a prop? Is it a style? Is it necessary?